src/js.rs

```diff
@@ -20,14 +22,17 @@ enum RngSource {
 
 // JsValues are always per-thread, so we initialize RngSource for each thread.
 //   See: https://github.com/rustwasm/wasm-bindgen/pull/955
-thread_local!(
+#[cfg(std)]
+std::thread_local!(
     static RNG_SOURCE: Result<RngSource, Error> = getrandom_init();
 );
 
-pub(crate) fn getrandom_inner(dest: &mut [MaybeUninit<u8>]) -> Result<(), Error> {
-    RNG_SOURCE.with(|result| {
-        let source = result.as_ref().map_err(|&e| e)?;
+#[cfg(not(std))]
+#[thread_local]
```

Comment:
`#[thread_local]` is unstable and even after stabilization will have a very high MSRV.

Reply:
Yep, you're right however if I'm not mistaken this is the only alternative for no_std envs